packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,24 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let filterDotFiles = function (x) {
+    return x[0] === ".";
+  };
```

Comment:
I'm a little confused here, the default value is a filter which [excludes dot files](https://github.com/fs-utils/fs-readdir-recursive/blob/master/index.js#L9)